permits/filters.py

```diff
@@ -56,6 +56,30 @@ class OwnPermitRequestFilterSet(BasePermitRequestFilterSet):
     pass
 
 
+def _get_administrative_entities_queryset_for_current_user(request):
+    return (
+        models.PermitAdministrativeEntity.objects.filter(
+            works_object_types__permit_requests__in=services.get_permit_requests_list_for_user(
```

Comment:
There is also a relation between `PermitAdministrativeEntity` and permit requests in via the `permit_requests` related name, so line 62 should be:
`permit_requests__in=services.get_permit_requests_list_for_user(`